internal/store/deployments.go

```diff
@@ -30,6 +33,36 @@ type HostDescriber interface {
 	Host(string, string) (*opsmngr.Host, error)
 }
 
+type HostDatabaseLister interface {
```

Comment:
I don't think this should be part of this PR. You don't have to change it, it is just my opinion since this doesn't seem related to this scope of the PR.

Reply:
the reason I did it was to group the related methods and thus reducing the files to tag with go generate, if you check they all  use the same `opsmngr. Deployments` module, this changed on the client but we never updated the file grouping so here I'm doing that, but to your point yes, they could've been a separate PR